### PR TITLE
Translations for gahuza marathi burmese

### LINKED ISSUE #5661: Update translations for Gahuza,  Marathi and Burmese
**Is your feature request related to a problem? Please describe.**
Editorial have supplied an update to the translations for the above sites
**Describe the solution you'd like**
Update the translation files

**Describe alternatives you've considered**
None.

**Testing notes**
Check translations in Storybook match the ones described below

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc
No

- [ ] This feature is expected to need manual testing.
Yes
**Additional context**

**Marathi**

Link | English | Tranlsation
-- | -- | --
Trust Project link | Why you can trust the BBC | तुम्ही बीबीसीवर विश्वास ठेवू शकता कारण
Links in footer | Privacy policy | गोपनीयतेचं धोरण
Links in footer | Contact the BBC | बीबीसीशी संपर्क साधा
Copyright message in footer | © 2019 BBC. | © 2019 BBC. बीबीसी बाह्य इंटरनेट साइट्सच्या सामग्रीसाठी बीबीसी जबाबदार नाही. बाह्य लिंक्सबद्दल आम्हाल काय वाटतं? इथे वाचा.

**Gahuza**

Site | English | Gahuza
-- | -- | --
Nav around site |   |  
  | Video | Video
  | Audio | Amajwi
  | Listen | Umviriza
  | Watch | Raba
  | Previous radio show | Ikiganiro ca radiyo giheruka
  | Next radio show | Ikiganiro ca radiyo gikurikira
  | Duration | Umwanya bimara
Media player (pages) |   |  
  | Audio Player | Kina amajwi
  | Video Player | Kina amasanamu
  | Media Player | Ibikinwa

**Burmese**

Error   pages |   |  
-- | -- | --
  | Double   checking the url | စာမျျက်နှာ   လင့် လိပ်စာကို ထပ်စစ်ကြည့်နေပါသည်။
  | Internal   server error | ဘီဘီစီဆာဗာ၏   အမှား
NAVIGATION   AROUND SITE |  
  | Watch | ကြည့်ရှုရန်
  | LIVE | တိုကျရိုကျထုတျလှှငျ့မှု
  | Previous   radio show | ယခင်   ရေဒီယိုအစီအစဉ်
  | Next   radio show | နောက်   ရေဒီယိုအစီအစဉ်
  | Duration | ကြာမြင့်ချိန်
Footer |   |  
  | Contact the BBC | ဘီဘီစီကို   ဆက်သွယ်ရန်
MEDIA   PAGES (ERROR MESSAGES) |  
  | To   play this content, please enable JavaScript, or try a different browser | သငျ့စကျတှငျ   ဒီမီဒီယာဖိုငျကို ဖှငျ့၍ မရပါ။

### PR DESCRIPTION
**Pull request by JonBeeb**
Resolves #5661 

**Overall change:** 
Added translations. 
 **Code changes:**
- _A bullet point list of key code changes that have been made._
- _When describing code changes, try to communicate **how** and **why** you implemented something a specific way, not just **what** has changed._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/lib/config/services/marathi.js [hunk 1]
```diff
@@ -165,7 +165,7 @@ export const service = {
     footer: {
       trustProjectLink: {
         href: 'https://www.bbc.com/marathi/institutional-50418391',
-        text: 'तुम्ही बीबीसीवर विश्वास का ठेवू शकता',
+        text: 'तुम्ही बीबीसीवर विश्वास ठेवू शकता कारणा',
```
Comment (OlgaLyubin) [inline src/app/lib/config/services/marathi.js:169]: That last word seems a bit different than in translations sheet " कारण "

File: src/app/lib/config/services/marathi.js [hunk 2]
```diff
@@ -178,19 +178,19 @@ export const service = {
...
+          text: 'गोपनीयतेचं धोरण',
         },
         {
           href: 'https://www.bbc.com/usingthebbc/cookies/',
           text: 'कुकीज',
         },
         {
           href: 'https://www.bbc.com/marathi/institutional-42227676',
-          text: 'बीबीसीला संपर्क करा',
+          text: 'बीबीसीशी संपर्क साधाा',
```
Comment (OlgaLyubin) [inline src/app/lib/config/services/marathi.js:190]: Same with this one, seems to have an extra line at the end  साधा

Comment (itimirichard) [reply src/app/lib/config/services/marathi.js:190]: @JonBeeb Is the translation update here intentional? from` बीबीसीला संपर्क करा` to `बीबीसीशी संपर्क साधाा`

Comment (JonBeeb) [reply src/app/lib/config/services/marathi.js:190]: Yes. That is what they have supplied in the spreadsheet.

File: src/app/lib/config/services/marathi.js (hunk 1 above)
Comment (mmcculloch) [reply src/app/lib/config/services/marathi.js:169]: It's a little weird, if I CMD+F it matches both, even though I can see the difference. Might be something to do with how these languages are implemented I don't quite understand.

File: src/app/lib/config/services/marathi.js (hunk 2 above)
Comment (mmcculloch) [reply src/app/lib/config/services/marathi.js:190]: image

Like the other one it seems to look different sometimes, I'm not entirely sure why.

File: src/app/lib/config/services/marathi.js [hunk 3]
```diff
@@ -178,19 +178,19 @@ export const service = {
...
         },
       ],
       copyrightText:
-        'बीबीसी बाह्य इंटरनेट साइट्सच्या सामग्रीसाठी बीबीसी जबाबदार नाही',
+        'बीबीसी बाह्य इंटरनेट साइट्सच्या सामग्रीसाठी बीबीसी जबाबदार नाही.',
```
Comment (mmcculloch) [inline src/app/lib/config/services/marathi.js:194]: The text in the issue is longer 
```
© 2019 BBC. बीबीसी बाह्य इंटरनेट साइट्सच्या सामग्रीसाठी बीबीसी जबाबदार नाही. बाह्य लिंक्सबद्दल आम्हाल काय वाटतं? इथे वाचा.
```
Is it just supposed to be this part?

File: src/app/lib/config/services/marathi.js (hunk 1 above)
Comment (DenisHdz) [reply src/app/lib/config/services/marathi.js:169]: I can see the same text across the issue description, PR and translation sheet: 
`तुम्ही बीबीसीवर विश्वास ठेवू शकता कारण`

File: src/app/lib/config/services/marathi.js (hunk 2 above)
Comment (DenisHdz) [reply src/app/lib/config/services/marathi.js:190]: They all look the same to me 🤷🏻‍♂️

File: src/app/lib/config/services/marathi.js (hunk 3 above)
Comment (DenisHdz) [reply src/app/lib/config/services/marathi.js:194]: The other half is the `externalLink` text.

Comment (DenisHdz) [reply src/app/lib/config/services/marathi.js:194]: They are combined in the Psammead Sitewide Link component:
```
<StyledParagraph>
    {copyrightText}{' '}
    <Link text={externalLink.text} href={externalLink.href} inline />
</StyledParagraph>
```

**Review by DenisHdz — APPROVED** [review]
LGTM 👍

File: src/app/lib/config/services/marathi.js (hunk 2 above)
Comment (OlgaLyubin) [reply src/app/lib/config/services/marathi.js:190]: @DenisHdz it's a tiny difference of an extra vertical line at the end. Not sure if it's a problem, just wanted to be sure.

Screenshot 2020-03-04 at 10 56 47
Screenshot 2020-03-04 at 10 56 39

File: src/app/lib/config/services/marathi.js (hunk 1 above)
Comment (jimjohnsonrollings) [reply src/app/lib/config/services/marathi.js:169]: This is most likely to do with how clever the software rendering the text is.  In some languages, two characters when next to one another combine into a single glyph.  This is like the old English a and e combining to æ in words like encyclopædia. So I think it's fine.

File: src/app/lib/config/services/marathi.js (hunk 2 above)
Comment (jimjohnsonrollings) [reply src/app/lib/config/services/marathi.js:190]: This is most likely to do with how clever the software rendering the text is.  In some languages, two characters when next to one another combine into a single glyph.  This is like the old English a and e combining to æ in words like encyclopædia. So I think it's fine.

**Review by OlgaLyubin — APPROVED** [review]
LGTM

**Pull request merged**